### ISSUE TITLE
export MetaTagsConfig interface

### ISSUE THREAD

**Issue by zanona** [lordgiotto/react-metatags-hook#12]
Just a small thing here, really, but it would be great to import interfaces in case we wanted to create a custom wrapper for generating `MetaTagsConfig`

at the moment, this is what I'm doing:

```tsx
import useMetaTags from 'react-metatags-hook';
type MetaTagsConfig = Parameters<typeof useMetaTags>[0];
type PageInfo = Partial<{title: string; description: string; image: string; path: string}>;
type PageMeta = (info:PageInfo) => MetaTagsConfig;
``` 
and here's what I'd like to do:

```tsx
import type {MetaTagsConfig} from 'react-metatags-hook';
type PageInfo = Partial<{title: string; description: string; image: string; path: string}>;
type PageMeta = (info:PageInfo) => MetaTagsConfig;
```

**Comment by lordgiotto** [comment]
Hi @zanona I just released the new version 1.3.1 that exports `MetaTag`, `LinkTag` and `MetaTagsConfig` types.
Let me know if this works for you.
Cheers :)

**Comment by zanona** [comment]
Thanks a lot. That works nicely! :+1:
